apps/shell/utils/HostAllowlist.js

```diff
@@ -13,7 +13,11 @@ class HostAllowlist {
         .map(yml => yaml.safeLoad(fs.readFileSync(yml)))
         .filter(config => (config.v2 && config.v2.login && config.v2.login.host) && ! (config.v2 && config.v2.metadata && config.v2.metadata.hidden))
         .map(config => {
-            return { host: config.v2.login.host, default: config.v2.login.default }
+            let isDefault = false;
+            if (config.v2.login.default){ // a catch in case the default is undefined for cluster configs
+                isDefault = true
+            }
+            return { host: config.v2.login.host, default: isDefault }
```

Comment:
Using a ternary operator here would improve readability:
```suggestion
            return { host: config.v2.login.host, default: config.v2.login.default ? true : false }
```

Reply:
I would prefer that `utils/HostAllowlist.js` be renamed to be `utils/host-allowlist.js` for readability and to keep capitalization out of file names. Make sure to update the import in `app.js` and `allowlist.test.js` to be:

```js
const HostAllowlist = require('./utils/host-allowlist');
```